index.html

```diff
@@ -22,31 +22,34 @@
   <body>
     <div class="page-container">
       <div class="header1">
-      <header class="headercontainer">
-        <a href="#" class="logo">Megha</a>
-        <nav><i class="fa-solid fa-bars hambrgricon"></i></nav>
-      </header>
-    </div>
-<div class="header2">
-      <header class="headercontainer">
-        <a href="#" class="logo">Megha</a>
-        <nav><ul class="navlist">
-          <li> <a href="#">Portfolio</a></li>
-          <li> <a href="#">About</a></li>
-          <li> <a href="#">Contact</a></li>
-        </ul></nav>
-      </header>
-    </div>
+        <header class="headercontainer">
+          <a href="#" class="logo">Megha</a>
+          <nav><i class="fa-solid fa-bars hambrgricon" id="hmbg"></i></nav>
+        </header>
+      </div>
+      <div class="header2">
+        <header class="headercontainer">
+          <a href="#" class="logo">Megha</a>
+          <nav>
+            <ul class="navlist">
+              <li class="linkitems"><a href="#portfolio">Portfolio</a></li>
+              <li class="linkitems"><a href="#about">About</a></li>
+              <li class="linkitems"><a href="#contact">Contact</a></li>
+            </ul>
+            <i class="fa-solid fa-times closeicon" id="closebtn"></i>
+          </nav>
```

Comment:
The nav menu in the desktop mode should not be closeable. Right now you have a close button which upon pressing, closes the navbar. Kindly fix this issue.
![image](https://user-images.githubusercontent.com/33268927/181473577-63281dc7-77ee-4996-9a96-1fe95f6c426a.png)